pkg/provider/azure_managedDiskController.go

```diff
@@ -211,7 +213,8 @@ func (c *ManagedDiskController) CreateManagedDisk(options *ManagedDiskOptions) (
 	})
 
 	if err != nil {
-		klog.V(2).Infof("azureDisk - created new MD Name:%s StorageAccountType:%s Size:%v but was unable to confirm provisioningState in poll process", options.DiskName, options.StorageAccountType, options.SizeGB)
+		klog.Warningf("azureDisk - created new MD Name:%s StorageAccountType:%s Size:%v but was unable to confirm provisioningState in poll process, wait 3s for disk provisioning complete", options.DiskName, options.StorageAccountType, options.SizeGB)
+		time.Sleep(3 * time.Second)
```

Comment:
should this wait be part of the above `wait.ExponentialBackoff`, e.g. L207?

Reply:
I will change this `wait.ExponentialBackoff` logic later in next commit, when it's in `GetDisk` throttling, `wait.ExponentialBackoff` would make things worse, imagine create 1K disks in parallel, it will get throttled all time.
Will skip `wait.ExponentialBackoff` if it's already throttled, and only blind sleep 3s in `GetDisk` throttling case.